packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -128,10 +128,18 @@ describe("TableComponent", () => {
 		element.dispatchEvent(clickEvent);
 	};
 
+	const triggerHover: Function = (element: HTMLElement): void => {
+		// more verbose way to create and trigger an event (the only way it works in IE)
+		// https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events
+		const hoverEvent: MouseEvent = document.createEvent("MouseEvent");
+		hoverEvent.initEvent("mouseenter", true, true);
+		element.dispatchEvent(hoverEvent);
+	};
+
 	const DUMMY_DATA: object[] = [
-		{ id: 1, description: "dummy 1" },
```

Comment:
Were this whitespaces actually removed by Prettier? That would be weird :thinking: